addons/medical_gui/stringtable.xml

```diff
@@ -1,11 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project name="ACE">
     <Package name="Medical GUI">
-        <Key ID="STR_ACE_Medical_GUI_Category">
-            <English>ACE Medical GUI</English>
-            <German>Sanitätsmenü</German>
-            <Japanese>ACE 医療 GUI</Japanese>
-            <Russian>ACE Медицина - Интерфейс</Russian>
+        <Key ID="STR_ACE_Medical_GUI_SubCategory">
+            <English>GUI</English>
```

Comment:
```suggestion
            <English>Medical GUI</English>
```

Reply:
Actually, I guess this is under medical subcategory already? In which case ignore